bin/truffleruby

```diff
@@ -120,7 +138,28 @@ if [ -n "$CP" ]; then
 fi
 
 declare -a full_command
-full_command=("$JAVACMD" $JAVA_OPTS "-Xbootclasspath/a:$boot_cp_jar" "${java_args[@]}" org.truffleruby.Main "-Xhome=$root" "-Xlauncher=$root/bin/truffleruby" "${ruby_args[@]}")
+if [ $on_graalvm = false ]; then
+    full_command=(
+        "$JAVACMD"
+        $JAVA_OPTS
+        "-Xbootclasspath/a:$boot_cp_jar"
+        "${java_args[@]}"
+        org.truffleruby.Main
+        "-Xhome=$root"
+        "-Xlauncher=$root/bin/truffleruby"
+        "${ruby_args[@]}"
+    )
+else
+    full_command=(
+        $JAVACMD # no " to let options expand to multiple array elements
+        $JAVA_OPTS
+        "${java_args[@]}"
+        com.oracle.graalvm.Main
+        org.truffleruby.Main
+        "-Xlauncher=$root/bin/truffleruby"
+        "${ruby_args[@]}"
+    )
+fi
```

Comment:
I'd rather have one file then more, we've already forgot to carry over a fix. Anyway there is not much time we can revisit.